net/socket/transport_client_socket_unittest.cc

```diff
@@ -296,6 +296,7 @@ TEST_F(TransportClientSocketTest, FullDuplex_ReadFirst) {
   // write callback has executed.  We wait for the read callback to run now to
   // make sure that the socket can handle full duplex communications.
 
+  LOG(INFO)<<"waiting for callback.WaitForResult()";
```

Comment:
Are those changes still needed, or are they more development aids ? 

Preferably the upstream Chromium code in `//net` and `//base` should be modified as little as possible.

Reply:
Agreed. They should be removed.